WeakAuras/RegionTypes/Icon.lua

```diff
@@ -509,7 +509,11 @@ local function modify(parent, region, data)
   function region:UpdateSize()
     local width = region.width * math.abs(region.scalex);
     local height = region.height * math.abs(region.scaley);
-
+    local glowing;
+    if region.glow then
+      region:SetGlow(false);
```

Comment:
I briefly discusses this with @Dooez in discord today. He suggested that the best way is to fix the show method of the glow to take size changes into account.

So we don't need to hide the glow here.